libiocage/lib/Jail.py

```diff
@@ -153,6 +245,20 @@ def force_stop(self):
         return successful
 
     def create(self, release_name, auto_download=False):
+        """
+        Create a Jail from a Release
+
+        Args:
+
+            release_name (string):
+                The jail is created from the release matching the name provided
+
+            auto_download (bool): (default=False)
```

Comment:
Good catch, haha! This can be implemented on CLI side, so no need to expose an interface to automatically fetch the release here. Thanks!